test/runner/lib/cloud/vmware.py

```diff
@@ -0,0 +1,120 @@
+"""Vmware plugin for integration tests."""
```

Comment:
This functionality should go into the existing `vcenter.py` plugin. Use an environment variable to override the current behavior (using the simulator) and use this new logic instead.